src/input/ytdl_src.rs

```diff
@@ -14,7 +14,13 @@ use std::{
 use tokio::task;
 use tracing::trace;
 
+#[cfg(feature = "youtube-dl")]
+const YOUTUBE_DL_COMMAND: &str = "youtube-dl";
+#[cfg(feature = "youtube-dlc")]
+const YOUTUBE_DL_COMMAND: &str = "youtube-dlc";
```

Comment:
Can you make this so that the default is `"youtube-dl"`, and the `youtube-dlc` feature changes this?

E.g., constant computation allows constructs such as:
```rust
const TEST: &str = if cfg!(feature = "") {
    "A"
} else {
    "B"
};
```